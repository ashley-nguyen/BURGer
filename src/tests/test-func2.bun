func testPrint(int a, double b, char c, string d){
	print(a);
	print(b);
	print(c);
	print(d);
	}
testPrint(4, 3.54, "c", "hello");


