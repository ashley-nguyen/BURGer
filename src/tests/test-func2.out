4
3.54
c
hello
